projects/1.20/assets/butchery/butcher/lang/zh_cn.json

```
@@ -0,0 +1,709 @@
+{
+  "advancements.breathless.descr": "谁会不喜欢气球呢？即便是用青蛙做的...",
```

```suggestion
  "advancements.breathless.descr": "谁会不喜欢气球呢？即便是用青蛙做的……",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。